index.html

```diff
@@ -624,113 +624,111 @@ <h4>
     this specification.
   </p>
 
-  <aside title="Push a text string to either a tag or peer"
-       class="example">
+  <section><h3>Push a text string to either a tag or peer</h3>
     <p>
       Pushing a text string to any kind of device is straightforward.
       Options can be left out, as they default to pushing to both tags
       and peers.
     </p>
-    <pre class="highlight">
-const writer = new NFCWriter();
-writer.push(
-  "Hello World"
-).then(() => {
-  console.log("Message pushed.");
-}).catch(error => {
-  console.log(`Push failed :-( try again: ${error}.`);
-});
+    <pre class="example">
+      const writer = new NFCWriter();
```

Comment:
Doesn't this extra indentation add extra indentation to the spec?

Reply:
No, I compared it with the original, no difference at the rendered doc.